specsuite/Readme.md

```diff
@@ -0,0 +1,36 @@
+# Specsuite Documentation
```

Comment:
Could you rename the file from `Readme.md` to `README.md`? This would be consistent with the naming of the readme in the repo root.

Reply:
Pushed the rename of `Readme.md` to `README.md`